src/compas_rhino/geometry/plane.py

```diff
@@ -92,9 +50,19 @@ def from_selection(cls):
     def to_compas(self):
         """Convert to a COMPAS geometry object.
 
+        Returns
+        -------
+        :class:`compas.geometry.Plane`
+            A COMPAS plane.
+        """
+        return plane_to_compas(self.geometry)
+
+    def to_compas_frame(self):
```

Comment:
Should these be `staticmethod`s instead? Because they don't really construct the object in which they are located, but they are very useful and clearly related.

A related comment: in `compas_fab` we added methods like `coerce_frame` , which take as argument either a compas `Frame` or a rhino `Plane` and return a compas `Frame`. Do we have a similar thing now in this PR that can return the compas object of X regardless of whether the input is already of that type?

Reply:
> Should these be `staticmethod`s instead? Because they don't really construct the object in which they are located, but they are very useful and clearly related.

how will this work with `self.geometry`?

Reply:
> A related comment: in `compas_fab` we added methods like `coerce_frame` , which take as argument either a compas `Frame` or a rhino `Plane` and return a compas `Frame`. Do we have a similar thing now in this PR that can return the compas object of X regardless of whether the input is already of that type?

this is indeed the case for, for example, `RhinoPlane.from_geometry` which takes both a Rhino plane and a COMPAS frame